dbms/src/Columns/ColumnUnique.h

```diff
@@ -300,19 +300,21 @@ StringRef ColumnUnique<ColumnType>::serializeValueIntoArena(size_t n, Arena & ar
 {
     if (is_nullable)
     {
-        const UInt8 null_flag = 1;
-        const UInt8 not_null_flag = 0;
+        static constexpr UInt8 null_flag = 1;
+        static constexpr UInt8 not_null_flag = 0;
+        static constexpr auto s = sizeof(null_flag);
 
         auto pos = arena.allocContinue(sizeof(null_flag), begin);
-        auto & flag = (n == getNullValueIndex() ? null_flag : not_null_flag);
-        memcpy(pos, &flag, sizeof(flag));
+        auto flag = (n == getNullValueIndex() ? null_flag : not_null_flag);
+        memcpy(pos, &flag, s);
```

Comment:
We probably don't need three declarations for 0, 1 and 1, that are only used once.

```
auto null_flag = reinterpret_cast<UInt8 *>(arena.allocContinue(sizeof(UInt8), begin));
*null_flag = (n == getNullValueIndex()) ? 1 : 0;
```

Reply:
Ok

Reply:
Now `flag` is `int` and memcpy is wrong for some endianness. Also I'd write `unalignedStore`, because I always forget why we have to do memcpy. Anyway, alignof(char) == 1 so it's always aligned and we can just dereference.

Reply:
Yes. Of course I need to use `UInt8`. Thank you!